client/cmd/apply/apply.go

```diff
@@ -187,6 +195,7 @@ func (c *applyCommand) printFailed(namespaceName, operation, kind, name, cause s
 
 func (c *applyCommand) printFailedAll(operation, kind, cause string) {
 	c.logger.Error("[all] %s: %s %s ❌", operation, kind)
+	c.errors.Append(fmt.Errorf("[all] %s: %s %s ❌", operation, kind, cause))
```

Comment:
same as the previous comment, appending errors can be put outside of printFailed